SpecializedServers/MultiSocks/DirtySocks/Messages/PersOut.cs

```diff
@@ -9,9 +9,9 @@ public class PersOut : AbstractMessage
         public string? EXticker { get; set; }
         public string? IDLE { get; set; }
         public string? LA { get; set; }
+        public string LKEY { get; set; } = "3fcf27540c92935b0a66fd3b0000283c";
```

Comment:
In pair with the request, the LKEY must be null and only attributed for older clients. btw PSINCE must stay, it is used in the sims.

Reply:
Hindsight on my part, I didn't mean to remove PSINCE, but for LKEY, this is static atm for testing purposes from VTS-Tech, and seems to work fine in practice. What's your definition of "older clients" as so far I have no issues with this string in LKEY for any game.

Reply:
Actually current way is sub-optimal I agree. We can't filter per clients version. If it causes no problems it's fine then. But a better system will be developped.